src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/AttributeComponent.scala

```diff
@@ -140,12 +141,17 @@ trait AttributeComponent {
           val attributeRecsWithRefForName: Set[(AttributeRecord, Option[EntityRecord])] = attributeRecsWithRefForNameWithDupes.map { case ((wsId, attributeRec), entityRec) => (attributeRec, entityRec) }.toSet
           if (attributeRecsWithRefForName.forall(_._1.listIndex.isDefined)) {
             unmarshalList(attributeRecsWithRefForName)
-          } else if (attributeRecsWithRefForName.size > 1) {
-            throw new RawlsException(s"more than one value exists for attribute but list index is not defined for all, records: $attributeRecsWithRefForName")
-          } else if (attributeRecsWithRefForName.head._2.isDefined) {
-            unmarshalReference(attributeRecsWithRefForName.head._2.get)
           } else {
-            unmarshalValue(attributeRecsWithRefForName.head._1)
+            //HACK: Work around corrupted data due to GAWB-443. If there are duplicate records for this attribute,
+            //log an error (not an exception) and then just take the first one.
+            if (attributeRecsWithRefForName.size > 1) {
+              logger.error(s"more than one value exists for attribute but list index is not defined for all records: $attributeRecsWithRefForName size: ${attributeRecsWithRefForName.size} numEmpty: ${attributeRecsWithRefForName.count(_._1.listIndex.isEmpty)}")
+            }
+            if (attributeRecsWithRefForName.head._2.isDefined) {
```

Comment:
I think we should sort it if there are more than one so that we predictably choose the same one each time